src/ray/core_worker/core_worker.cc

```diff
@@ -1487,10 +1487,10 @@ Status CoreWorker::CreateActor(const RayFunction &function,
   // Add the actor handle before we submit the actor creation task, since the
   // actor handle must be in scope by the time the GCS sends the
   // WaitForActorOutOfScopeRequest.
-  std::unique_ptr<ActorHandle> actor_handle(new ActorHandle(
+  auto actor_handle = std::make_unique<ActorHandle>(
```

Comment:
only to make sure it's working

Reply:
If we support c++14 we can get more benefits, we can optimize performance more easier.
We can use lambda capture move in c++14:
```c++
  std::string str = "test";
  //c++11, we have to copy here.
  auto lambda11 = [str]{
  };

  //c++14, move the str, no copy anymore
  auto lambda14 = [value = std::move(str)]{
  };
```

You can also use generic lambda  and auto function to simplify your code:
```c++
//generic lambda, it's very useful to simplify template code.
auto lambda = [](auto x, auto y) {return x + y;};

///auto function to cut the tail in c++11

//in c++11
template<typename F, typename Tuple, int... I>
auto apply_(F&& f, Tuple&& args, int_seq<I...>) ->
  decltype(std::forward<F>(f)(std::get<I>(std::forward<Tuple>(args))...)) //auto-decltype tail in c++11
  {
    return std::forward<F>(f)(std::get<I>(std::forward<Tuple>(args))...);
  }

template <typename F, typename Tuple, size_t... I>
decltype(auto) apply_impl(F&& f, Tuple&& t, index_sequence<I...>) { //no tail anymore
    return forward<F>(f)(get<I>(forward<Tuple>(t))...);
}
```

There also some other cool features in c++14, such as template variable, more useful type traits, actually the ci has run on gcc 5.4 which has already fully supported c++14, i really hope we can update to c++14.

Today is 2021, c++20 published last year, but we are using the c++ standard 10 years ago, we should update at least c++14.

c++17 and c++20 are also very good to own:)